deployment/data-streams/changeset/verifier/v0_5_0/call_latest_config_details.go

```diff
@@ -0,0 +1,55 @@
+package v0_5_0
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+)
+
+var LatestConfigDetailsChangeset = deployment.CreateChangeSet(LatestConfigDetailsLogic, LatestConfigDetailsPrecondition)
+
+type LatestConfigDetailsConfig struct {
+	ConfigsByChain map[uint64][]LatestConfigDetails
+}
+
+type LatestConfigDetails struct {
+	VerifierAddress common.Address
+	ConfigDigest    [32]byte
+}
+
+func (cfg LatestConfigDetailsConfig) Validate() error {
+	if len(cfg.ConfigsByChain) == 0 {
+		return errors.New("ConfigsByChain cannot be empty")
+	}
+	return nil
+}
+
+func LatestConfigDetailsPrecondition(_ deployment.Environment, cc LatestConfigDetailsConfig) error {
+	if err := cc.Validate(); err != nil {
+		return fmt.Errorf("invalid LatestConfigDetails config: %w", err)
+	}
+	return nil
+}
+
+func LatestConfigDetailsLogic(e deployment.Environment, cfg LatestConfigDetailsConfig) (deployment.ChangesetOutput, error) {
```

Comment:
I don't think we actually need this changeset - since it's just a query more or less

Reply:
We do have [this sequence](https://github.com/smartcontractkit/gauntlet-plus-plus/blob/main/packages-ethereum/operations-mercury/src/sequences/validate-config-digests.sequence.ts#L33) in G++ using latest config details.
Do you know if it's being used?

Reply:
Yes we'll need something like that - ok I see the approach here now. We could create wrapper changeset that is `ValidateConfigDigests` which calls sub-changesets as needed. LGTM then - but let me know if this isn't the approach you were thinking